affirm_example/__init__.py

```diff
@@ -196,16 +227,20 @@ def shopping_item_page():
 
     default_currency = app.config.get('DEFAULT_CURRENCY', 'USD')
 
+    merchant_info = {'public_api_key': app.config["AFFIRM"]["PUBLIC_API_KEY"]}
+
+    url_config = kwargs.copy()
+    url_config.update(merchant_info)
```

Comment:
any reason we are adding this here? 
is it mainly because we want to make sure the `user_confirmation_url` always contain a `?public_api_key=xxx` ?

Reply:
correct!